### DISCUSSION TITLE
Is there a way to sort camera feed on frigate UI

### DISCUSSION THREAD

**Discussion by balajeek** [blakeblackshear/frigate#5190] [Ask A Question] [answered]
Under "Camera" menu from the UI, it shows all my camera feeds, but i like some cameras appear first, second and so in the order.
I thought i can put the camera in order in the config, but that did not help.  I search thru and couldn't find an answer.
any help would be appreciated.  thanks.

**Comment by NickM-27** [comment, marked as answer]
See the docs. For each camera you specify 

```
ui:
  order: 
```

To set the order in the ui

**Comment by balajeek** [comment]
that was super fast response, thank you.
I did search in the document using the search (looking for order, sort) but came back page not found.  Then after you said, i looked in configuration section and was able to see there.  (wonder why search wouldn't bring it up)

i did the ui/order and it works now.  thanks again.

**Comment by aage21** [comment]
do you have an example of what it should look like with camera names? default is 0. I dont understand how to list the cameras. It tells me it expects an integer. 

thanks

**Reply by NickM-27** [reply]
```
cameras:
  one_camera:
    ui:
      order: 2
  three_camera:
    ui:
      order: 1
```

**Reply by aage21** [reply]
ok, thanks. so you put ui: order under every camera and number them in the order you like.

thanks!

**Comment by mayerwin** [comment]
The config `ui: order` is the clean way to set the global order. If you'd rather drag to arrange (and also reorder the All Cameras view, which isn't draggable by default), I built a small companion that does that in the UI and saves it server-side, so the arrangement carries across your devices and browsers. Free and MIT-licensed: https://github.com/mayerwin/frigate-layout-sync
